libc/test/UnitTest/LibcTest.h

```diff
@@ -162,6 +162,14 @@ class Test {
                           (unsigned long long)RHS, LHSStr, RHSStr, Loc);
   }
 
+  // Helper to allow macro invocations like `ASSERT_EQ(foo, nullptr)`.
+  template <typename ValType,
+            cpp::enable_if_t<cpp::is_pointer_v<ValType>, ValType> = nullptr>
+  bool test(TestCond Cond, ValType LHS, std::nullptr_t, const char *LHSStr,
```

Comment:
Looks like this is breaking the buildbots.

```
/home/libc_worker/libc-riscv64-debian/libc-riscv64-debian-fullbuild-dbg/llvm-project/libc/test/UnitTest/LibcTest.h:168:41: error: use of undeclared identifier 'std'
  bool test(TestCond Cond, ValType LHS, std::nullptr_t, const char *LHSStr,
                                        ^
```

mind reverting if you can't fix forward quickly?

Reply:
We have libc/src/__support/CPP/type_traits/is_null_pointer.h which defines `cpp::nullptr_t`.  That include and that value should be used in place of `std::nullptr_t` I suspect.

Reply:
pull/114321 should fix it

Reply:
Thanks for helping with this forward fix, and apologies for the breakage :)